stacker/lookups/handlers/output.py

```diff
@@ -37,11 +37,5 @@ def handler(value, provider=None, context=None, fqn=False, **kwargs):
 
 
 def deconstruct(value):
-
-    # Probably an undefined environment variable
```

Comment:
I think this is still necessary, though now that I've looked at it an if statement was probably the wrong way to catch it.  Here's how it should probably be written (note, I think the `value.split` is wrong in and of itself as well:

```
try:
    stack_name, output_name = value.split("::", 1)
except ValueError:
    raise ValueError("output handler requires syntax of <stack>::<output>.  Got: %s" % value)
```

Not married to the message in the ValueError though, if you can think of something better